scripts/eurostat/health_determinants/fruits_vegetables/download.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 Google LLC
+#
```

Comment:
Please rename this file to different name; so that there is no confusion between this module and the one present in common folder.
Please refer to other Eurostat imports for how names are used.